settings.gradle

```diff
@@ -1 +1,17 @@
+pluginManagement {
+    repositories {
```

Comment:
Suggestion (💡): Consider moving as much of the `Plugin DSL` related configuration to the `settings.gradle` file. For example, you could end up with the below:
```
pluginManagement {
    gradle.ext.agpVersion = '7.2.1'

    plugins {
        id 'com.android.application' version gradle.ext.agpVersion
        id 'com.android.library' version gradle.ext.agpVersion
    }

    repositories {
        gradlePluginPortal()
        google()
        mavenCentral()
    }
}
```

Also, with the above you might consider removing the root level `build.gradle` build file as it is no longer necessary. PS: The `clean` task is not necessary and can be usually removed.

Reply:
Thanks! I've committed this change and removed the root project `build.gradle`. 

I'm not sure about `clean` task - now it doesn't remove the `build` directory of root project when running `clean`. But I think it's a minor thing.